### DISCUSSION TITLE
Foundations: Basic Computer Literacy Quiz

### DISCUSSION THREAD

**Discussion by aidqen** [TheOdinProject/curriculum#25688] [Ideas]
### Describe your suggestion

Hey!
I'm going through the foundations course currently and I realized I know a lot of the simple content about computers and it was irrelevant for me to go through it again.
So my suggestion was, when someone wants to go through the foundations course, have him take a quizz which will reveal his level of expertise in computers and will make his journey a lot more productive.
I can see how it could benefit many different people especially in the computer basics phase.
Thank you for listening!

### Path

Foundations

### Lesson Url

theodinproject.com/lessons/foundations-computer-basics

### Checks

- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [ ] Would you like to work on this issue?

### (Optional) Discord Name

aidqen#1676

### (Optional) Additional Comments

nope

**Comment by CouchofTomato** [comment]
Hey @aidqen 

I've converted this to a discussion because it isn't something we could action at this time but it can remain visible so others may comment if they feel it's a good idea.

I think the main issue would be the length of the quiz required in order for us to fairly take a call on where learners should start their learning.

**Reply by CBID2** [reply]
Hey @CouchofTomato. To address the issue length, maybe instead of a quiz, it could be a 5 question survey that provides recommendations for learning paths and/or specific lessons based on the learners' answers. It'll create a more personalized learning experience.

**Reply by CouchofTomato** [reply]
It would be very difficult for us to do that. Some of our lessons also make reference to earlier lessons for example. It would require a complete re-think and re-write of the architecture and lessons. I'm not sure this is viable at this time.

**Reply by CBID2** [reply]
Ooh interesting point. With this in mind, I guess the idea is no from me.
